IFC4x3/Properties/n/NominalWidth_2bcxE0qSOHuO00025QrE$V/DocProperty.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalWidth_2bcxE0qSOHuO00025QrEV" Name="NominalWidth" UniqueId="2bcxE0qSOHuO00025QrE$V" PrimaryDataType="IfcPositiveLengthMeasure">
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="NominalWidth" UniqueId="2bcxE0qSOHuO00025QrE$V" PrimaryDataType="IfcPositiveLengthMeasure">
 	<Localization>
 		<DocLocalization Locale="en" Name="Nominal Width" />
 		<DocLocalization Locale="ja-JP" Name="公称幅">
```

Comment:
As before can we keep the localization

Reply:
Yes, I have copied the localization in subsequent commit.